rules/no-ignored-test-files.js

```diff
@@ -0,0 +1,92 @@
+'use strict';
+var path = require('path');
+var pkgUp = require('pkg-up');
+var multimatch = require('multimatch');
+var util = require('../util');
+var createAvaRule = require('../create-ava-rule');
+
+var defaultFiles = [
+	'test.js',
+	'test-*.js',
+	'test/**/*.js'
+];
+
+function isInIgnoredFolder(relativeFilePath) {
+	var folders = relativeFilePath.split(path.sep);
+	var ignoredFolder;
+	if (folders.indexOf('fixtures') !== -1) {
+		ignoredFolder = 'fixtures';
+	} else if (folders.indexOf('helpers') !== -1) {
+		ignoredFolder = 'helpers';
+	}
+
+	if (ignoredFolder) {
```

Comment:
Why not use `multimatch` here as well? Copy [`excludePatterns`](https://github.com/sindresorhus/ava/blob/005b656b9950ab3fa1e2bd9af326d5b39540e71b/api.js#L34:L37) without the leading `!` and you should be good to go.